includes/service-providers/class-newspack-newsletters-service-provider.php

```diff
@@ -336,6 +336,24 @@ public function insert_post_data( $data, $postarr ) {
 		return $data;
 	}
 
+	/**
+	 * Notify the site administrators.
+	 *
+	 * @param string $subject The email subject.
+	 * @param string $message The email message.
+	 */
+	private static function notify_site_admins( $subject, $message ) {
```

Comment:
I think sending these emails as well as selecting how many and which accounts are notified should be tied to some option or hook to allow publishers to opt-in or modify this. Forcing every publisher to start receiving failure emails seems a bit extreme.

Reply:
Updated in cfa11e6255df7be8427ea4af6e44b11a3bde6ab0 to use the email listed as the `admin_email` option.